### ISSUE TITLE
Add reset values option

### ISSUE THREAD

**Issue by vv-monsalve** [Lorp/figma-colrv1#7]
This option would be great if you regret and want to take back the changes done into the palette options. `cmd + z` doesn't work.
But even if it worked, sometimes user would like to go back to defaults in a single step.

**Comment by Lorp** [comment]
Good point. Perhaps two reset buttons: one for the colours and one for axes.

**Comment by vv-monsalve** [comment]
Even better! Especially if you are going to include the CSS output

**Comment by Lorp** [comment]
For the axis values, now that the Default style is always available from the Style menu, I don’t think I’ll add any additional axis reset buttons. For the palettes, it’s still useful however.
